bitnami/airflow/templates/_helpers.tpl

```diff
@@ -348,20 +349,48 @@ Compile all warnings into a single message, and call fail.
+airflow: airflow.cloneDagFilesFromGit.repositories
+    At least one repository must be provided when enabling downloading DAG files
+    from git repository (--set airflow.cloneDagFilesFromGit.repositories[0].repository="xxx"
+    --set airflow.cloneDagFilesFromGit.repositories[0].name="xxx"
+    --set airflow.cloneDagFilesFromGit.repositories[0].branch="name")
+{{- end -}}
+{{- end -}}
+{{/* Validate values of Airflow - "airflow.cloneDagFilesFromGit.repositories.repository", "airflow.cloneDagFilesFromGit.repositories.name", "airflow.cloneDagFilesFromGit.repositories.branch" must be provided when "airflow.cloneDagFilesFromGit.enabled" is "true" */}}
+{{- define "airflow.validateValues.cloneDagFilesFromGit.repository_details" -}}
+{{- if .Values.airflow.cloneDagFilesFromGit.enabled -}}
+{{- range $index, $repository_detail := .Values.airflow.cloneDagFilesFromGit.repositories }}
+{{- if empty $repository_detail.repository -}}
+airflow: airflow.cloneDagFilesFromGit.repositories[$index].repository
     The repository must be provided when enabling downloading DAG files
-    from git repository (--set airflow.cloneDagFilesFromGit.repository="xxx")
+    from git repository (--set airflow.cloneDagFilesFromGit.repositories[$index].repository="xxx")
 {{- end -}}
+{{- if empty $repository_detail.name -}}
+airflow: airflow.cloneDagFilesFromGit.repositories[$index].name
+    The name must be provided when enabling downloading DAG files
+    from git repository (--set airflow.cloneDagFilesFromGit.repositories[$index].name="xxx")
 {{- end -}}
-{{/* Validate values of Airflow - "airflow.cloneDagFilesFromGit.branch" must be provided when "airflow.cloneDagFilesFromGit.enabled" is "true" */}}
-{{- define "airflow.validateValues.cloneDagFilesFromGit.branch" -}}
-{{- if and .Values.airflow.cloneDagFilesFromGit.enabled (empty .Values.airflow.cloneDagFilesFromGit.branch) -}}
-airflow: airflow.cloneDagFilesFromGit.branch
+{{- if empty $repository_detail.branch -}}
+airflow: airflow.cloneDagFilesFromGit.repositories[$index].branch
     The branch must be provided when enabling downloading DAG files
-    from git repository (--set airflow.cloneDagFilesFromGit.branch="xxx")
+    from git repository (--set airflow.cloneDagFilesFromGit.repositories[$index].branch="xxx")
+{{- end -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{/* Validate values of Airflow - If "airflow.cloneDagFilesFromGit.enabled" is "true" either "airflow.cloneDagFilesFromGit.repositories" should be provided or "airflow.cloneDagFilesFromGit.repository" and "airflow.cloneDagFilesFromGit.branch" both should be provided */}}
+{{- define "airflow.validateValues.cloneDagFilesFromGit.repository" -}}
+{{- if and .Values.airflow.cloneDagFilesFromGit.enabled (empty .Values.airflow.cloneDagFilesFromGit.repositories) -}}
+{{- if or (empty .Values.airflow.clonePluginsFromGit.repository) (empty .Values.airflow.clonePluginsFromGit.branch) -}}
```

Comment:
I would add this statement to `"airflow.validateValues.cloneDagFilesFromGit.repositories"` and remove this test. Otherwise, tests are going to have conflicts

Reply:
@dani8art Understood. Done.